docs/hugo/content/contributing/create-a-new-release.md

```diff
@@ -1,3 +1,6 @@
+---
+title: Creating a new release of ASO v2
+---
 # Creating a new release of ASO v2
```

Comment:
I haven't looked at the output yet but it might be that the title will be appearing in two locations on the page?

Reply:
… or maybe we should have the template do that already if it doesn’t, to make the source files easier to manage.

Reply:
Current theme (Book) leaves the page to do it's own thing - the FrontMatter is only used for sidebar/etc.

It seems more common to do things the other way around - Docsy injects the title at the top of the page, and has a separate FrontMatter item for the name in the sidebar.